### PR TITLE
[10.0] Migrate product_weight

### PR DESCRIPTION
**Pull request by florian-dacosta**

### CODE REVIEW

File: product_weight/README.rst [hunk 1]
```diff
@@ -19,9 +21,27 @@ You can update the weight of one product clicking on the button under
...
 
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/135/10.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/{project_repo}/issues>`_. In case of trouble, please
```
Comment (lasley) [inline product_weight/README.rst]: Bad link

File: product_weight/__manifest__.py [hunk 2]
```diff
@@ -1,37 +1,23 @@
...
+    'summary': 'Allows to calculate products weight from its components.',
+    'depends': [
+        'mrp',
+    ],
+    'data': [
+        'wizard/product_weight_update_view.xml',
+        'views/product_view.xml',
+    ],
+    'installable': True,
+    'images': [],
```
Comment (lasley) [inline product_weight/__manifest__.py:22]: Remove empty key

File: product_weight/models/bom.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    _inherit = 'mrp.bom.line'
+
+    @api.multi
+    def get_final_components(self):
+        bom_lines = []
+        for line in self:
+            if not line.child_line_ids:
+                bom_lines.append(line)
+            else:
+                bom_lines.extend(line.child_line_ids.get_final_components())
```
Comment (lasley) [inline product_weight/models/bom.py]: Just FYI, inplace is more efficient than `.extend` due to no method call:
```
bom_lines += line.child_line_ids.get_final_components()
```

**Review by lasley — APPROVED** [review]
Thanks @florian-dacosta - few notes, nothing major

File: product_weight/__manifest__.py (hunk 2 above)
Comment (florian-dacosta) [reply product_weight/__manifest__.py:22]: I thought it should always be present?
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md
(in modules category)

**Comment by florian-dacosta** [comment]
Thanks for the review, I applied the changes

**Review by leemannd — APPROVED** [review]
Nothing to add to @lasley review

**Pull request merged**